src/pages/Lending/Pool/components/Borrow/BorrowConfirm.tsx

```diff
@@ -206,6 +179,9 @@ export const BorrowConfirm = ({
         cryptoAmount: depositAmount ?? '0',
         assetId: collateralAssetId,
         to: lendingQuoteData.quoteInboundAddress,
+        // We assume the funds are already in the right address at this point
+        // Since we're relying on a mempool Tx and not waiting for completion, we can't use a `from` address
+        // as the balance would not be confirmed yet
```

Comment:
no longer an applicable comment? we are still using the explicit `from` address to correctly pick the utxos associated with the `from` address (both confirmed and/or unconfirmed)

Reply:
ca9542b89f